src/pages/DivideTime/index.js

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect } from 'react'
+import { Link } from 'react-router-dom'
+
+import * as S from './style'
+
+import Button from 'shared/components/Button'
+import NavBar from 'shared/components/NavBar'
+import TaskBox from 'pages/DivideTime/components/TaskBox'
+import Text from 'shared/components/Text'
+import TimeSelectForm from 'shared/components/TimeSelectForm'
+import { convertSecondsToHourMinute } from 'shared/utils/convertTime'
+import { themeColors } from 'shared/constants/colors'
+import { usePendingTimers } from './hooks/usePendingTimers'
+import { useTimers } from '../../shared/hooks/useTimers'
+
+const BUTTON_TEXT = Object.freeze({
+	VALID: '다음 단계',
+	INVALID: '시간을 입력해주세요',
+})
+
+const DivideTime = () => {
+	const {
+		isTimeOver,
+		pendingTimers,
+		totalTime,
+		selectedTask,
+		initPendingTimers,
+		changeTime,
+		handlePendingTimerBoxClick,
+	} = usePendingTimers()
+
+	const { resetTimers } = useTimers()
+
+	useEffect(() => {
+		initPendingTimers()
+	}, [])
+
+	const handleSubmit = time => {
+		changeTime(time)
+	}
+
+	const handleNextPageClick = () => {
+		const newTimers = {}
+		pendingTimers.forEach(({ name, time, id }) => {
+			newTimers[id] = { time, name }
+		})
```

Comment:
요기도 고차함수로 한번에 반환해도 될 것 같네용 
```suggestion
		const newTimers  = Object.entries(timers).reduce((acc, [id, timer])  => {
			acc[id] = { time, name }
			return acc
		},{})
```